### DISCUSSION TITLE
How could I add "my favourite" to this repository ?

### DISCUSSION THREAD

**Discussion by febfour** [community/community#3872] [Mobile]
Though I searched this repository, but couldn't find this repository.
This repository is not private...why?

repository:
https://github.com/clover-network

**Comment by nplasterer** [comment]
Hi @febfour I believe what you linked is an organization? Are non of this orgs repositories showing up? 
We are trying to improve the search experience so that problems like this don't happen. Would having an add to favorites button on the repository itself help?
